app/assets/stylesheets/map/leaflet.css.scss

```diff
@@ -245,6 +245,33 @@
 	border: 5px solid #bbb;
 	}
 
+.leaflet-draw-section {
+	position: relative;
+	top: 110px;

```

Comment:
Something like `top: 11px; left: 57px;` might be better?

![bbg-leaflet-draw-position](https://cloud.githubusercontent.com/assets/1818302/17109857/c7b65c06-5267-11e6-8d80-5eb3b06262f2.png)